service/lib/agama/storage/config_conversions/from_model_conversions/config.rb

```diff
@@ -53,21 +54,24 @@ def default_config
           # @return [Hash]
           def conversions
             drives = convert_drives
+            raids = convert_raids
+            partitionables = Array(drives) + Array(raids)
 
             {
-              boot:          convert_boot(drives || []),
+              boot:          convert_boot(partitionables),
```

Comment:
Can we assign a MD as boot device? Right now the solver only considers drives as candidates for boot device.

Reply:
Discussed in a planning.